index.html

```diff
@@ -72,8 +91,8 @@ <h2 class="playfairFont projectsTag">Projects</h2>
         </section>
 
         <section class="flex-column">
-          <img class="fullWidth" src="resources/ABlaptop.png" alt="A laptop and a phone">
-          <div class="darkBlueBackground projectsPadding flex-column">
+          <img class="fullWidth ABLaptopPic" src="resources/ABlaptop.png" alt="A laptop and a phone">
```

Comment:
- [ ] You were so close, but as shown in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23-9&t=STzDZgG5tHGW7XUO-0) the width of this image should take around 50% of the screen's width, and the mac book image above should stick above it. Kindly make sure you fix it ✔. 
     - Figma design:
        ![image](https://user-images.githubusercontent.com/104968236/229290516-45cc915f-d04d-4041-aad6-784f053f82bb.png)
     - Your design:
        ![Inkedfirefox_ENU6HMj2nd](https://user-images.githubusercontent.com/104968236/229290628-1723f206-66ce-4bbd-b2eb-9f9283ae76ab.jpg)